### ISSUE TITLE
experimental rules checkbox not working?

### ISSUE THREAD

**Issue by cprice404** [nbadal/ktlint-intellij-plugin#120]
I enabled the experimental rules in my gradle config, and the gradle build now fails in places where I am violating them as I would expect.  However, I also checked the "Use experimental rules" checkbox in the IDEA plugin config dialog, and the errors that are causing the compile failures are not being highlighted in the editor window. Is there another setting I need to toggle?

**Comment by nbadal** [comment]
@cprice404 would you happen to know which rules it wasn't picking up? I can look into this!

**Comment by cprice404** [comment]
I noticed it for at least a couple:

* Argument list wrapping
* No leading empty lines in method blocks

I didn't see any editor highlighting when I violated those, but when I ran ktlint outside the IDE it definitely detected them.

I think I also tried and observed it on:

* Braces required for multiline if/else statements

**Comment by nbadal** [comment]
@cprice404 I haven't been able to reproduce this issue since you reported it – has it been happening for you still?

One idea: there are times where code analyzing _as a whole_ is slow or interrupted at times and nothing gets linted (along with other analyzers, not just mine). I see this occasionally on larger projects, but it usually resolves itself over time or with a restart of the IDE.

**Comment by adapap** [comment]
Here to +1 that experimental rules don't seem to be working (at least for `argument-list-wrapping`). Here is a repro case:

```kotlin
package com.samsara.remoteaccess

fun test(p1: String, p2: String, p3: String) {
  return
}

class Example {
  constructor() {
    test(
      "long argument for example call", "long argument for example call",
      "long argument for example call"
    )
  }
}
```

Error from ktlint: `Example.kt:10:41: Argument should be on a separate line (unless all arguments can fit a single line)`

No error caught by plugin (experimental rules enabled). It does actually format if all three arguments are on the same line, maybe this has to do with maximum line length configuration?

**Comment by nbadal** [comment]
@adapap thanks for the repro case! I'll take another look at this

**Comment by nbadal** [comment]
Confirmed that something has changed with regard to how ktlint likes being loaded via the class loader since I originally implemented it. Working on a fix now!

**Comment by nbadal** [comment]
It ended up being a shadow jar issue! Fixed in 0.9.0+
